README.md

```diff
@@ -511,10 +536,50 @@ If a `spend` fails because the amount spent exceeds the amount received the
 server MUST return a `402 Payment Required` response code and MUST NOT add to
 the spent amount.
 
+#### Authentication of the Sending System
+
+If the sending system/web monetization provider was identified during session
+creation, by providing a JWT as a Bearer token on the request, then the
+signature of the JWT MUST be validated AND the `jti` claim in the token MUST be
```

Comment:
Confusing as before we say that the wallet MAY track the spending against a provider and then here we are using MUST. Seems conflicting or we should just say that this applies only IF the wallet chooses to support aggregation